dpe/src/dpe_instance.rs

```diff
@@ -28,6 +28,7 @@ pub struct DpeEnv<'a, T: DpeTypes + 'a> {
     pub platform: T::Platform,
 }
 
+#[repr(C, align(4))]
```

Comment:
This should be zerocopy AsBytes/FromBytes as well; repr(C) doesn't stop people from putting values like enums in here that have "impossible" bit representations.

Reply:
Thanks to Zach's suggestions this looks a bit nicer now.